baseapp-content-feed/baseapp_content_feed/graphql/mutations.py

```
@@ -0,0 +1,57 @@
+        fields = ("content",)
+
+class ContentPostCreate(RelayMutation):
+    content_post = graphene.Field(ContentPostObjectType._meta.connection.Edge)
+    
+    class Input:
+        content = graphene.String(required=True)
+
+    @classmethod
+    @login_required
+    def mutate_and_get_payload(cls, root, info, **input):
+        content = input.get("content")
```

Ruff (0.7.0)

Local variable `content` is assigned to but never used

Remove assignment to unused variable `content`

(F841)